include/config.h

```diff
@@ -19,6 +19,12 @@
 #include "config/config_objects.h"
 #include "config/config_rom.h"
 
+// Local config - include a gitignore'd config file that's specific to just the user (if the file exists)
+#ifdef __has_include
```

Comment:
is this necessary

Reply:
Eh, the online example I saw had it. Probably a portability thing idk

Reply:
i believe all gcc versions 5.X and higher support it and the example i saw was from like 2017 so i think its pretty safe

Reply:
Alright I'll just tank it as long as it works in GCC9